src/components/meeting/Meeting.js

```diff
@@ -3,23 +3,32 @@ import {useParams} from "react-router-dom";
 import {interviewAPI} from "../../utils/apis/InterviewAPI";
 import {MeetingDisplayName} from "./MeetingDisplayName";
 import {JitsiWrapper} from "./JitsiComponentWrapper";
+import {CircularProgress} from "@material-ui/core";
 
 export const Meeting = () => {
     let { interviewId, companyId } = useParams()
     const [name, setName] = useState(null)
+    const [fetching, setFetching] = useState(true)
```

Comment:
Jeśli ta zmienną jest boolem to nie wiem czy konwencją nie jest początku z isFetching albo w elixirze byłoby fetching?

Reply:
wait what?

Reply:
Że nazwa zmiennej skoro to jest Boolean to powinno być isFetching

Reply:
a to tak, sorry, nie ogarnąłem komentarza wyżej XDD